hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -1996,7 +1995,7 @@ public void testLinkedAndNonLinkedBucketMetaData()
     execute(ozoneShell, new String[] {"bucket", "list", "/volume1"});
 
     // Expect valid JSON array
-    final ArrayList<LinkedTreeMap<String, String>> bucketListLinked =
+    final List<HashMap<String, Object>> bucketListLinked =
```

Comment:
If we can change to Map,  then kindly check for test cases as well.

Reply:
Done!